npm-shrinkwrap.json

```diff
@@ -3500,258 +3500,320 @@
       "integrity": "sha512-cM+1OnKRacBh98mw5RJ3PEmFxhl70lDFAL50+x/a+fxzSZryBF1asADf57zliAtGhCNVHYzrJAZX+8R77AGYGg==",
       "dependencies": {
         "abbrev": {
-          "version": "https://registry.npmjs.org/abbrev/-/abbrev-1.0.9.tgz",
+          "version": "1.0.9",
+          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.0.9.tgz",
```

Comment:
> I mended them by hand. Yup.

Ouch! :confused: 

> Haven't found out yet if this is npms fault,

If it is, I guess a small reproducible example would be useful.

> or if we're doing something in the pre/post-shrinkwrap scripts that causes this.

Would including the changes from #1514 help with this, or is that something else entirely?